src/Workspaces/Core/Portable/CodeActions/CodeAction.cs

```diff
@@ -222,15 +222,19 @@ protected virtual Task<Document> GetChangedDocumentAsync(CancellationToken cance
         /// <returns>A document with the post processing changes applied.</returns>
         protected virtual async Task<Document> PostProcessChangesAsync(Document document, CancellationToken cancellationToken)
         {
-            document = await Simplifier.ReduceAsync(document, Simplifier.Annotation, cancellationToken: cancellationToken).ConfigureAwait(false);
+            if (document.SupportsSyntaxTree)
```

Comment:
how about inverting if condition so that chunk of code doesn't need to be in { }